tests/applications/validation/class/missing_decorator_call/test_missing_all.py

```diff
@@ -0,0 +1,123 @@
+import unittest
+
+from tensorlake.applications import application, cls, function
+from tensorlake.applications.function.introspect import ClassDetails, FunctionDetails
+from tensorlake.applications.validation import (
+    ValidationMessage,
+    ValidationMessageSeverity,
+    validate_loaded_applications,
+)
+
+
+@cls
+class MyClass:
+    @application
+    @function
+    def my_method(self, payload: int) -> None:
+        pass
+
+
+class TestMissingDecoratorCall(unittest.TestCase):
+    def test_applications_fail_validation(self):
+        validation_messages: list[ValidationMessage] = validate_loaded_applications()
+        self.assertTrue(
+            len(validation_messages) == 3,
+        )
+
+        for validation_message in validation_messages:
+            self.assertEqual(
+                validation_message.severity,
+                ValidationMessageSeverity.ERROR,
+            )
+
+            if "@cls" in validation_message.message:
+                self.assertEqual(
+                    validation_message.message,
+                    "Class is missing () after @cls decorator. Please use @cls() syntax.",
```

Comment:
I'm concerned that people will see `missing ()` as an incomplete message because it's not usual to see parenthesis mentioned that way. Can these messages have a consistent format? A small suggestion:

```
The @cls decorator is missing its parenthesis. Please replace it with `@cls()`.
The @application decorator is missing its parenthesis. Please replace it with `@application()`.
The @function decorator is missing its parenthesis. Please replace it with `@function()`.
```

Reply:
Applied this comment with a couple changes:
* Didn't use `` because I don't use them in other messages.
* Didn't use "The" article because I don't use it in other messages.

This can be changed if we need to though (required updating 20+ tests).
I preferred consistency, this is why and was too lazy to change all tests.